todolist/app.go

```diff
@@ -174,30 +179,21 @@ func (a *App) UnprioritizeTodo(input string) {
 	fmt.Println("Todo un-prioritized.")
 }
 
-func (a *App) getId(input string) (int, *Todo) {
+func (a *App) getId(input string) int {
```

Comment:
I love this refactor.  it always bothered me that `getId` would return the todo.